torchaudio/models/_wavernn.py

```diff
@@ -5,101 +5,84 @@
 
 
 class _ResBlock(nn.Module):
-    r"""This is a ResNet block layer. This layer is based on the paper "Deep Residual Learning
-    for Image Recognition". Kaiming He,  Xiangyu Zhang, Shaoqing Ren, Jian Sun. CVPR, 2016.
-    It is a block used in WaveRNN. WaveRNN is based on the paper "Efficient Neural Audio Synthesis".
-    Nal Kalchbrenner, Erich Elsen, Karen Simonyan, Seb Noury, Norman Casagrande, Edward Lockhart,
-    Florian Stimberg, Aaron van den Oord, Sander Dieleman, Koray Kavukcuoglu. arXiv:1802.08435, 2018.
+    r"""ResNet block based on "Deep Residual Learning for Image Recognition"
+
+    The paper link is https://arxiv.org/pdf/1512.03385.pdf. The input signal is spectrogram.
 
     Args:
-        num_dims: the number of compute dimensions in the input (default=128).
+        n_freq: the number of bins in a spectrogram (default=128)
 
-    Examples::
-        >>> resblock = _ResBlock(num_dims=128)
+    Examples
+        >>> resblock = _ResBlock()
         >>> input = torch.rand(10, 128, 512)
-        >>> output = resblock(input)
+        >>> output = resblock(input) # shape: (10, 128, 512)
```

Comment:
nit: it's conventional to put two spaces before an inline comment
```
output = resblock(input)  # shape: (10, 128, 512)
```

Reply:
Updated.
I also updated this change in #735 and #724 .